rasa/core/test.py

```diff
@@ -955,7 +1023,8 @@ async def _evaluate_core_model(model: Text, stories_file: Text) -> int:
     agent = Agent.load(model)
     generator = await _create_data_generator(stories_file, agent)
     completed_trackers = generator.generate_story_trackers()
-    story_eval_store, number_of_stories = await _collect_story_predictions(
+
+    story_eval_store, number_of_stories, _ = await _collect_story_predictions(
```

Comment:
why don't we get entity results here?

Reply:
This method is just returning the number successful stories. Should we also incorporate the entities into this calculation?

Reply:
in this case, I'd say no. Could you please add a comment?